### PR TITLE
fix: change css and add some cards

### PR DESCRIPTION
**Pull request by sagargse**
### Some cards and buttons added

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -251,6 +251,7 @@ <h2>Primary Buttons</h2>
                                         class="fa fa-cloud-upload"></i></span></a>
                             <a class="primary-button icontext-button"><span class="btn-icon"><i
                                         class="fa fa-sign-in"></i></span>Sign-in</a>
+                            <a class="primary-button floating-button"><i class="fa fa-plus my-float"></i></a>
```
Comment (Shailseen) [inline documentation.html]: why are you using anchor tag i think you need an button.

**Review by Shailseen — APPROVED** [review]
all code looks good to me. you can proceed for merge

**Review by Rockyraven — APPROVED** [review]
your code is clean
 you can good to go

File: documentation.html (hunk 1 above)
Comment (sagargse) [reply documentation.html]: ooh okay

**Comment by sagargse** [comment]
> your code is clean you can good to go

thanks

**Pull request merged**